src/EscapeAnalysis.jl

```diff
@@ -323,6 +317,15 @@ function find_escapes(ir::IRCode, nargs::Int)
     return state
 end
 
+function update_non_bitstype_changes!(args::Vector{Any}, ir::IRCode, @nospecialize(info::EscapeInformation), changes::Changes)
```

Comment:
`info::EscapeInformation` can certainly lead to dynamic dispatch when `info` can't be uninferred. Not specializing it will solve the issue.